cpp/modmesh/view/R3DWidget.hpp

```diff
@@ -46,24 +46,22 @@
 namespace modmesh
 {
 
-class RScene
-    : public Qt3DCore::QEntity
+class RScene : public Qt3DCore::QEntity
```

Comment:
Did clang-format force you to make it in one line: `class RScene : public Qt3DCore::QEntity`?

If not, please change it back to use the two lines and avoid this unnecessary change of code.